libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/NBGeocoderMetadataParser.h

```diff
@@ -0,0 +1,19 @@
+//
+//  NBGeocoderMetadataParser.h
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "SQLiteDatabaseConnection.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NBGeocoderMetadataParser : NSObject
+
+- (instancetype)init;
```

Comment:
Remove? init is part of NSObject already

Reply:
done.